packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncConfiguration.kt

```diff
@@ -283,6 +312,24 @@ public interface SyncConfiguration : Configuration {
         public fun errorHandler(errorHandler: SyncSession.ErrorHandler): Builder =
             apply { this.errorHandler = errorHandler }
 
+        /**
+         * Sets the strategy that would handle the client reset by this synced Realm. In the case that
+         * no strategy is defined it would use the default one defined in [AppConfiguration].
+         *
+         * @param syncClientResetStrategy custom strategy to handle client reset.
+         */
+        public fun syncClientResetStrategy(syncClientResetStrategy: SyncClientResetStrategy): Builder =
```

Comment:
```suggestion
        public fun syncClientResetStrategy(resetStrategy: SyncClientResetStrategy): Builder =
```

Maybe just `resetStrategy` or `strategy` , repeating the name like this gets a bit repetitive if you are using named parameters.